### PR TITLE
feat: add Google Analytics

### LINKED ISSUE #29: feat: add google analytics
### Problem

No analytics

### Suggested solution

Add analytics

### Alternatives considered

Google analytics

### Additional context

I just added some env variables with the values of the GA-4. It will automatically pick the right one depending on the environment

image

### PR DESCRIPTION
**Pull request by fairlighteth**
# Description

- Adds Google Analytics in the Docusaurus config
- Addresses https://github.com/cowprotocol/docs/issues/29 

# Related issues

Fixes #29

### CODE REVIEW

**Comment by fairlighteth** [comment]
Tested the Vercel preview deploy with https://tagassistant.google.com/ and confirmed working:
Screenshot 2024-01-10 at 12 12 34

File: docusaurus.config.ts
Comment (alfetopito) [inline docusaurus.config.ts:1]: While I'm all for the style that you are using, this repo does not follow it, so there's a lot of noise in the PR.
Can you revert the styling changes so only the analytics settings are visible?

Comment (alfetopito) [reply docusaurus.config.ts:1]: Nevermind, I see this one now https://github.com/cowprotocol/docs/pull/277

**Review by mfw78 — APPROVED** [review]

File: docusaurus.config.ts [hunk 1]
```diff
@@ -25,179 +27,191 @@ const config: Config = {
...
-      }
-    ]
+        },
+      },
+    ],
+    [
+      "@docusaurus/plugin-google-gtag",
+      {
+        trackingID: "G-FC7FBG6T06",
+        anonymizeIP: true,
```
Comment (anxolin) [inline docusaurus.config.ts:128]: it would be good to point out the line that changed (too many changes)

**Review by anxolin — COMMENTED** [review]
Nice!

**Pull request merged**